scripts/EN/Harry

```diff
@@ -1431,36 +1431,36 @@ LoadBackground(0xa); // 0x1738: 00001069
 LoadEffect(0xe, 0x1, 0x0); // 0x173e: 0000306d
 DrawBackground(); // 0x1748: 00000065
 PlayMusic(0x65, 0x1); // 0x174c: 0000206c
-ShowText((1026)"I begin to run. Only the sounds of my footsteps echo in the surroundings. Just where in the world has Cheryl gone? Did someone lead her away? Various concerns run through my mind."); // 0x1754: 0000100c
+ShowText((1026)"I begin to run. ¥nW2Only the sounds of my footsteps echo in the surroundings.¥nW2Just where in the world has Cheryl gone?¥nW2Did someone lead her away?¥nW2Various concerns run through my mind."); // 0x1754: 0000100c
 HandleInput(); // 0x175a: 00000010
 LoadSecondEffect(0x18, 0x1, 0x1); // 0x175e: 0000306e
 PlaySecondSoundFx(0x9); // 0x1768: 00001073
-ShowText((1027)"(Ta ta ta ta...) (Onomatopoeia for footsteps) I suddenly get the feeling that I heard footsteps other than my own, so I stop moving right away to check it out."); // 0x176e: 0000100c
+ShowText((1027)"(Tap tap tap tap…)¥n¥nW4I suddenly get the feeling that W1I heard footsteps other than my own, so I stop moving right away to check it out."); // 0x176e: 0000100c
```

Comment:
I changed the literal translation of the Japanese onomatopoeia to:

"Tap tap tap tap..."